lighthouse-cli/bin.ts

```diff
@@ -62,7 +62,8 @@ const cliFlags = yargs
     'list-trace-categories',
     'config-path',
     'perf',
-    'port'
+    'port',
+    'timeout'
```

Comment:
Thinking about this now, I wouldn't think `timeout` was the whole run, and there are better tools to accomplish that if you do want to limit that on the command line.

The only other thing I would guess as a user of this would be that `timeout` would apply to the entirety of each pass, as in not just page load but (∑`beforePass()`es + page load + ∑`afterPass()`es), to make sure a particular pass doesn't go on forever. As long as our gatherers are relatively bounded, though, these are roughly the same thing.

(I guess custom gatherers could go crazy, but that's on them)

I think I'm good with going with `--timeout`, but since it's such prime flag real estate (and not necessarily self-descriptive) maybe @paulirish and @ebidel want to weigh in.